src/ceph-disk/ceph_disk/main.py

```diff
@@ -2362,17 +2362,18 @@ def create_key(self):
         cluster = self.args.cluster
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
-        command_check_call(
+        command_with_stdin(
```

Comment:
could you add a `Conflicts:` section after your `Signed-off-by` line, explaining why it's not cherry-picked from master?